packages/tinacms/src/plugins/fields/BlocksFieldPlugin.tsx

```diff
@@ -143,21 +142,18 @@ const Blocks = function({ tinaForm, form, field, input }: BlockFieldProps) {
       <GroupListPanel>
         <ItemList>
           <Droppable droppableId={field.name} type={field.name}>
-            {(provider, snapshot) => (
+            {(provider) => (
               <div ref={provider.innerRef} className="edit-page--list-parent">
                 {items.length === 0 && <EmptyState />}
                 {items.map((block: any, index: any) => {
                   const template = field.templates[block._template]
                   if (!template) {
                     // TODO: if no template return invalid entry
                   }
-                  const itemProps = React.useCallback(
-                    (item: object) => {
-                      if (!template.itemProps) return {}
-                      return template.itemProps(item)
-                    },
-                    [template.itemProps]
-                  )
+                  const itemProps = (item: object) => {
+                    if (!template.itemProps) return {}
+                    return template.itemProps(item)
+                  };
```

Comment:
I removed this `useCallback` because it violates the rules of hooks and having it nested this deeply seems super sketch.

Reply:
Oh jeez! Good call I didn't even notice that it 😬